modules/cnf-understanding-workload-hints.adoc

```diff
@@ -62,3 +62,122 @@ perPodPowerManagement: true
     | Allows for power management per pod.
 
 |===
+
```

Comment:
Folks, we intentionally never included this in the docs. Are we sure we want so much detail in the main documentation?

Reply:
To be honest I am not sure and I don't know who makes a final call on that. Is a KB a better place, I knida tagged this under https://github.com/openshift/openshift-docs/pull/71797 I know @aireilly is looking at this but I think from reorg of TOC at the moment. In terms of content we need to look at improving that and @djuran raised this and I felt it improved the docs but maybe too much info. Hard to know

Reply:
The feedback we've received from NEP partner is that this level of detail is indeed what we need.  
The level of detail currently available is not sufficient for them to make an informed decision on which hints to use for their workload.

Reply:
IMO: It is going to be impossible to make this information accurate or keep it accurate so will just cause more confusion for our customers. These options change release to release and even within a release (through maintenance releases).